classes/submission/SubmissionAgencyVocab.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @file classes/submission/SubmissionAgencyVocab.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionAgencyVocab
+ *
+ * @brief Operations for retrieving and modifying a submission's assigned agencies
+ */
+
+namespace PKP\submission;
+
+use APP\facades\Repo;
+use Illuminate\Database\Eloquent\Builder;
+use PKP\controlledVocab\ControlledVocab;
+use PKP\core\PKPApplication;
+
+
+class SubmissionAgencyVocab extends ControlledVocab
```

Comment:
Do you think we could get rid of these classes entirely? If all we're doing is providing a convenience `getAgencies` function, rather than `withSymbolic(SOME_AGENCIES_CONSTANT, ...)`, then I don't think it's worth the extra classes.

Reply:
All of there classes have been removed . By removing these extra classes and few related others , our code also reduced significantly .